.github/workflows/release.yml

```diff
@@ -4,9 +4,6 @@ name: Release
 
 on:
   workflow_dispatch:
-  push:
-    branches:
```

Comment:
no

Reply:
bc it gets rid of the whole point of having branch protection

Reply:
no it doesn't?